src/os/ObjectStore.h

```diff
@@ -705,6 +705,15 @@ class ObjectStore {
     std::map<std::string, ceph::buffer::list> *out ///< [out] Returned keys and values
     ) = 0;
 
+#ifdef WITH_SEASTAR
```

Comment:
could you move this change into "os/bluestore: add BlueStore::omap_get_values()"?